llvm/lib/Target/X86/CMakeLists.txt

```diff
@@ -118,3 +118,6 @@ add_subdirectory(Disassembler)
 add_subdirectory(MCA)
 add_subdirectory(MCTargetDesc)
 add_subdirectory(TargetInfo)
+add_subdirectory(lab3)
+
+set(LLVM_TEST_DEPENDS ${LLVM_TEST_DEPENDS} PARENT_SCOPE)
```

Comment:
Is this change really required?

Reply:
At least as far as I understand, yes, otherwise plugin won't compile, I tried. We done the same in all previous labs. I don't know how Artem was able to build it without this dependency and he doesn't know either.